cli/package.json

```diff
@@ -54,8 +54,8 @@
     "@types/gradient-string": "^1.1.2",
     "@types/inquirer": "^9.0.2",
     "@types/node": "^18.8.0",
-    "prettier": "^2.7.1",
-    "prettier-plugin-tailwindcss": "0.2.0-beta.3",
+    "prettier": "^2.8.0",
+    "prettier-plugin-tailwindcss": "^0.2.0",
```

Comment:
i wanted to use satisfies but didn't work.

upgraded these to get the parsing correct